src/doc/trpl/enums.md

```diff
@@ -28,6 +28,16 @@ In `Character`, for instance, `Digit` gives a meaningful name for an `i32`
 value, where `Other` is only a name. However, the fact that they represent
 distinct categories of `Character` is a very useful property.
 
+We use the `::` syntax to use the name of each variant: They’re scoped by the name
+of the `enum` itself. This allows both of these to work:
+
+```rust,ignore
+Character::Digit(10);
+Hand::Digit;
+```
+
+Both variants are named `Digit`, but they’re scoped to the `enum` name.
+
```

Comment:
I think  these lines should be put here rather than before.
the before splite the explainations of ` equality operators`.